components/about/StatementDetail.tsx

```diff
@@ -1,16 +1,22 @@
-import React from 'react'
+import React from "react";
 
 interface PropsInterface {
-    className?: string
-    info: string
+  className?: string;
+  info: string;
+  number: string;
```

Comment:
kaning statement detail wala nani gi use anywhere else? like what if ato ni i reuse pero wala lay number? or ari rajud ni sa tong Statement nga component?

Reply:
i believe wala na siya gi use anywhere. unsaon para dili needed i prop drill ang number?

Reply:
oks raman i prop drill, pero i optional prop siya IF ever gi use siya anywhere. do u think nga in the future magamit tani nga component anywhere else?

Reply:
ahhh ic ic, para nako i don't ma use ni na component na elsewhere

Reply:
ok resolved